pkg/status/dist/templates/header.tmpl

```diff
@@ -19,7 +19,7 @@
 
   Clocks
   ======
-    {{- if .ntpOffset }}NTP offset: {{.ntpOffset}} s{{- end }}
+    {{if .ntpOffset }}NTP offset: {{.ntpOffset}}s{{- end }}
```

Comment:
you could spread the statement on 3 lines:

```
    {{- if .ntpOffset }}
    NTP offset: {{.ntpOffset}} s
    {{- end }}
```

There might be a better way, but this would work